pkg/snapstore/abs_snapstore.go

```diff
@@ -113,7 +114,8 @@ func NewABSSnapStore(config *brtypes.SnapstoreConfig) (*ABSSnapStore, error) {
 		domain = *absCreds.Domain
 	}
 
-	blobServiceURL, err := ConstructBlobServiceURL(absCreds.StorageAccount, domain)
+	emulatorEnabled := config.IsEmulatorEnabled || absCreds.EmulatorEnabled
+	blobServiceURL, err := ConstructBlobServiceURL(absCreds.StorageAccount, domain, emulatorEnabled)
```

Comment:
```suggestion
	blobServiceURL, err := ConstructBlobServiceURL(absCreds.StorageAccount, domain, config.IsEmulatorEnabled || absCreds.EmulatorEnabled)
```

Reply:
I deliberately used a separate variable `emulatorEnabled` for better readability.